### PR TITLE
make fetchai default crypto, update end-to-end tests

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

This PR changes the default crypto to fetchai. It also updates the end-to-end tests to run on new Agentland.

## Fixes

-

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -203,9 +208,9 @@
 DUMMY_CONNECTION_PUBLIC_ID = PublicId("dummy_author", "dummy", "0.1.0")
 DUMMY_SKILL_PUBLIC_ID = PublicId("dummy_author", "dummy", "0.1.0")
 
-MAX_FLAKY_RERUNS = 3
-MAX_FLAKY_RERUNS_ETH = 1
-MAX_FLAKY_RERUNS_INTEGRATION = 1
+MAX_FLAKY_RERUNS = 0  # 3
```
Comment (DavidMinarsch) [inline tests/conftest.py:211]: Revert before merging!

File: tests/test_docs/test_cli_vs_programmatic_aeas/test_cli_vs_programmatic_aea.py [hunk 2]
```diff
@@ -172,12 +172,13 @@ def _inject_location(self, location, dst_file_path):
         """Inject location into the weather client strategy."""
         file = Path(dst_file_path)
         lines = file.read_text().splitlines()
+        line_insertion_position = 163  # line below: `strategy._is_ledger_tx = False`
```
Comment (DavidMinarsch) [inline tests/test_docs/test_cli_vs_programmatic_aeas/test_cli_vs_programmatic_aea.py:175]: @marcofavorito fyi

File: aea/configurations/base.py [hunk 3]
```diff
@@ -1783,7 +1783,7 @@ def _get_contract_interfaces(self) -> Dict[str, str]:
                 with open(full_path, "r") as interface_file_ethereum:
                     contract_interface = json.load(interface_file_ethereum)
                     contract_interfaces[identifier] = contract_interface
-            elif identifier == "cosmos":
+            elif identifier in ["cosmos", "fetchai"]:
```
Comment (marcofavorito) [inline aea/configurations/base.py:1786]: (not related to this PR): Eventually, also these magic values should be replaced.

**Review by marcofavorito — APPROVED** [review]
LGTM

File: aea/configurations/base.py (hunk 3 above)
Comment (DavidMinarsch) [reply aea/configurations/base.py:1786]: yes..there is a circular import here if I replace now..we need to clean it

**Review by solarw — APPROVED** [review]
LGTM

**Pull request merged**